pkg/scheduler/server/api.go

```diff
@@ -179,7 +179,7 @@ func (s *Server) triggerJob(ctx context.Context, req *api.TriggerRequest) bool {
 	}); err != nil {
 		// TODO: add job to a queue or something to try later this should be
 		// another long running go routine that accepts this job on a channel
-		log.Errorf("Error sending job to connection stream: %s", err)
+		log.Debugf("Error sending job %s to connection stream: %s", req.GetName()[idx+2:], err)
```

Comment:
qq: If there are errors sending jobs to dapr, is it retried in place or is currently dropped in the alpha implementation?

Reply:
Please see the [failure policy proposal PR lines for the answer here](https://github.com/dapr/proposals/pull/66/files#diff-eabc9eb37f9fcfc26ba06ecae4e6391a5a4ebda446a2bc8f85178f042609ef6bR14-R15) :)